book/visual-effects.md

```diff
@@ -409,12 +406,46 @@ can do the same and more with Skia and a few lines of Python.
 Skia is also the font library
 =============================
 
-Note that I've also replaced Tkinter's `measure` call with Skia's
-`measureText` equivalent. We'll also need to create Skia fonts instead
-of Tkinter ones. Update all the other places that `measure` was called
-to use `measureText`. We also need to change everywhere `metrics` is
-called to instead use Skia's `getMetrics`, which works a little
-differently. In Skia, ascent and descent are accessible via:
+Since we're replacing `tkinter`, we are also replacing `tkinter.font`.
+Luckily, Skia has an internal font cache, so we no longer need to
```

Comment:
Did you verify that?

Reply:
No—we should check this. If it's not true we can stick the cache back in.

Reply:
To clarify, I'm pretty confident it is. `Typeface` objects have a `uniqueID` method and if you create three identical fonts they all return the same `uniqueID`.